README.md

```diff
@@ -46,12 +46,12 @@ You can find more information and talk to developers
 
 ## Getting Started
 
-GeoTrellis is currently available for Scala 2.11 and Spark 2.0+.
+GeoTrellis is currently available for Scala 2.11 and 2.12, using Spark 2.4+.
 
 To get started with SBT, simply add the following to your build.sbt file:
 
 ```scala
-libraryDependencies += "org.locationtech.geotrellis" %% "geotrellis-raster" % "1.1.0"
+libraryDependencies += "org.locationtech.geotrellis" %% "geotrellis-raster" % "2.2.0"
```

Comment:
GeoTrellis `2.3.1` is the most recent version. So mb "using Spark 2.4.x" and "2.3.1" version?

Reply:
ok done